src/backend/expungeservice/endpoints/oeci_login.py

```diff
@@ -52,7 +52,7 @@ def post(self):
             secure=os.getenv("TIER") == "production",
             httponly=False,
             samesite="strict",
-            expires=time.time() + 15 * 60, # type: ignore # 15 minutes
+            expires=time.time() + 2 * 60 * 60, # type: ignore # 2 hour lifetime
```

Comment:
Please include a comment in the commit message about why it should be 2 hours (referring to the life time of the OECI session).

Reply:
Cool, done.